<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamineDlg</class>
 <widget class="QDialog" name="ExamineDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Examine Image</string>
  </property>
  <widget class="QTDisplay" name="Display1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>601</width>
     <height>261</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="AbortButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ContinueButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTDisplay</class>
   <extends>QWidget</extends>
   <header location="global">qtdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ContinueButton</sender>
   <signal>clicked()</signal>
   <receiver>ExamineDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
